[IMP] accounting/l10n_cl: Warning for automation disabling (#13062)
odoo/documentation: content/applications/finance/fiscal_localizations/chile.rst

   :alt: Digital certificate configuration.
   :align: center

.. warning::

>    If the :guilabel:`Certificate Owner` field is set for a specific user and no certificate 
>    is shared with all users, the automatic sending of electronic documents and receipt 
>    acknowledgments will be **disabled**.

Reviewer: Just a slight reword to avoid using "will"
```suggestion
   If the :guilabel:`Certificate Owner` field is set for a specific user, and there are no
   certificates shared with users, then the automatic sending of electronic documents and receipt
   acknowledgments is **disabled**.
```

> .. warning::
>    If the :guilabel:`Certificate Owner` field is set for a specific user, and there are no
>    certificates shared with users, then the automatic sending of electronic documents and receipt
>    acknowledgments is **disabled**.
> 

Reviewer 2: ```suggestion
.. warning::
   If the :guilabel:`Certificate Owner` field is set to a specific user, and there are no
   certificates shared with users, then the automatic sending of electronic documents and receipt
   acknowledgments is **disabled**.

```
"set for a specific user" makes it sound like each user has their own settings here and it can be set for one user and not set for another user. From my understanding (please correct me if I'm wrong), setting a user here means that this certificate is limited to the specified user, and if another user also has an exclusive certificate, they would need a separate one. If this is true, "set **to** a specific user" is more clear.

Author: Perfect! @samueljlieber is it possible you can help me with this last change?

Thanks!
